draft-ietf-quic-multipath.md

```diff
@@ -1232,6 +1264,63 @@ a PATH_RESPONSE frame in order to indicate the preferred path usage
 before or during path initiation.
 
 
+## MP_NEW_CONNECTION_ID frames {#mp-new-conn-id-frame}
+
+An endpoint sends a MP_NEW_CONNECTION_ID frame (type=0x15228c09) instead of 
+the NEW_CONNECTION_ID frame to provide its peer with alternative connection IDs 
+that can be used to break linkability when migrating connections; 
+see {{Section 19.15 of QUIC-TRANSPORT}}.
+
+MP_NEW_CONNECTION_ID frames are formatted as shown in {{fig-mp-connection-id-frame-format}}.
+
+~~~
+MP_NEW_CONNECTION_ID Frame {
+  Type (i) = 0x15228c09,
+  Sequence Number (i),
+  Retire Prior To (i),
+  Path Identifier (i),
```

Comment:
As the path identifier determines the sequence number space, I'd prefer to put it at the front.
```suggestion
  Path Identifier (i),
  Sequence Number (i),
  Retire Prior To (i),
```